snapcraft/extensions/gnome.py

```diff
@@ -73,7 +75,7 @@ def is_experimental(base: Optional[str]) -> bool:
     def get_app_snippet(self) -> Dict[str, Any]:
         return {
             # TODO: command-chain to check connection.
-            "command-chain": ["$SNAP/gnome-platform/desktop-launch"],
+            # "command-chain": ["$SNAP/gnome-platform/desktop-launch"],
```

Comment:
the error you see on
https://github.com/snapcore/snapcraft/runs/6677866843?check_suite_focus=true#step:4:170

is most likely due to this line